utilities/test_suite/REFERENCE_OUTPUT/tensor_max/tensor_max_u8_Tensor_PKD3.csv

```diff
@@ -0,0 +1,2 @@
+255,240,236,255,255,242,241,255,253,255,255,255
```

Comment:
We shouldn't use these csv files for defining costants. This can simply be repalced by const table in the code

Reply:
These are just the final golden outputs, Since for images these are large 50x50 or 100x100 elements x 3 channels, we had put golden outputs in files.

Since we wanted to maintain consistency for all functions, we were adding them in files with the thought that, at any point, that whole folder could be moved out of RPP parent repo if needed later.

But for small 10 or 12 element golden outputs like this function, we could have it declared in the test script header file itself? Please let us know

Reply:
In that case, can you please combine all the csv files into one. Every PR is adding a bunch of csv files for golden output. It will explode the number of csv files

Reply:
Used a map to store reference outputs